packages/sonarwhal/tests/lib/config.ts

```diff
@@ -270,3 +270,37 @@ test(`if a Rule has an invalid configuration, it should tell which ones are inva
 
     t.is(invalid.length, 1);
 });
+
+test('If formatter is specified as CLI argument, fromConfig method will use that to build SonarwhalConfig', (t) => {
+    const { config } = t.context;
+    const userConfig = {
+        connector: { name: 'chrome' },
+        formatters: ['summary', 'excel'],
+        rules: { 'apple-touch-icons': 'warning' }
+    } as UserConfig;
+    const cliOptions = { _: ['https://bing.com'], formatters: 'database' } as CLIOptions;
+
+    const result = config.SonarwhalConfig.fromConfig(userConfig, cliOptions);
+
+    t.true(result.formatters.length===1);
+    t.true(result.formatters[0]==='database');
+    // Make sure we updated only the formatters. Other properties of userConfig should stay same
+    t.true(result.connector.name==='chrome');
+});
+
+test('If formatter is not specified as CLI argument, fromConfig method will use the formatter specified in the userConfig object as it is to build SonarwhalConfig', (t) => {
+    const { config } = t.context;
+    const userConfig = {
+        connector: { name: 'chrome' },
+        formatters: ['summary', 'excel'],
+        rules: { 'apple-touch-icons': 'warning' }
+    } as UserConfig;
+    const cliOptions = { _: ['https://bing.com'] } as CLIOptions;
```

Comment:
Can you change `https://bing.com` to `https://example.com` just so that it's consistent with what we use in general. Thanks!

Reply:
Updated